[BUG] - image processing script makes the container crashes when an empty image is processed

### First Check

- [x] This is not a feature request.
- [x] I added a very descriptive title to this issue (title field is above this).
- [x] I used the GitHub search to find a similar issue and didn't find it.
- [x] I searched the Mealie documentation, with the integrated search.
- [x] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

I want to update the images previously uploaded to my mealie instance with the python script provdided [here](https://docs.mealie.io/documentation/getting-started/updating). But the script crashes when it processes empty files.

My instance is running in a docker container, and when the script has crashed, it brought the whole container with it.

### Steps to Reproduce

1. start mealie container
2. empty an image from an existing recipe
3. try to process it with the script
   ```bash
   python /opt/mealie/lib64/python3.12/site-packages/mealie/scripts/reprocess_images.py --workers 1
   ```

### Please provide relevant logs

```
root@c4cde70e3b2c:/# python /opt/mealie/lib64/python3.12/site-packages/mealie/scripts/reprocess_images.py --workers 1
INFO     2026-01-02T15:13:22 - Fetching recipes for image reprocessing
Found 60 recipes needing image reprocessing (using 1 worker). Proceed? (y/n) y
INFO     2026-01-02T15:13:27 - Starting image reprocessing...
ERROR    2026-01-02T15:13:27 - [ERROR] Failed to minify original.webp. Error: cannot identify image file '/app/data/recipes/d521132d-015c-4de5-93c9-5cb6a9fc8a9f/images/original.webp'
ERROR    2026-01-02T15:13:27 - [ERROR] Failed to minify original.webp. Error: cannot identify image file '/app/data/recipes/d521132d-015c-4de5-93c9-5cb6a9fc8a9f/images/original.webp'
ERROR    2026-01-02T15:13:27 - Failed to reprocess images for recipe d521132d-015c-4de5-93c9-5cb6a9fc8a9f
Traceback (most recent call last):
  File "/opt/mealie/lib64/python3.12/site-packages/mealie/scripts/reprocess_images.py", line 120, in process_recipe
    reprocess_recipe_images(recipe_id, force_all)
  File "/opt/mealie/lib64/python3.12/site-packages/mealie/scripts/reprocess_images.py", line 87, in reprocess_recipe_images
    service.minifier.minify(original_image, force=True)
  File "/opt/mealie/lib/python3.12/site-packages/mealie/pkgs/img/minify.py", line 171, in minify
    with Image.open(image_path) as img:
         ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/mealie/lib/python3.12/site-packages/PIL/Image.py", line 3560, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file '/app/data/recipes/d521132d-015c-4de5-93c9-5cb6a9fc8a9f/images/original.webp'
ERROR    2026-01-02T15:13:27 - Failed to reprocess images for recipe d521132d-015c-4de5-93c9-5cb6a9fc8a9f
Traceback (most recent call last):
  File "/opt/mealie/lib64/python3.12/site-packages/mealie/scripts/reprocess_images.py", line 120, in process_recipe
    reprocess_recipe_images(recipe_id, force_all)
  File "/opt/mealie/lib64/python3.12/site-packages/mealie/scripts/reprocess_images.py", line 87, in reprocess_recipe_images
    service.minifier.minify(original_image, force=True)
  File "/opt/mealie/lib/python3.12/site-packages/mealie/pkgs/img/minify.py", line 171, in minify
    with Image.open(image_path) as img:
         ^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/mealie/lib/python3.12/site-packages/PIL/Image.py", line 3560, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file '/app/data/recipes/d521132d-015c-4de5-93c9-5cb6a9fc8a9f/images/original.webp'
```

### Mealie Version

**Details**
Version: v3.9.1
Build: bde7cf6f9db63966879e725d5b2fd48a353b4f66
Mode de l’application: Production
Mode démo: Non
Port de l’API: 9000
Documentation de l’API: Activé
Type de base de données: sqlite
Foyer par défaut: Family
Version du Scraper de recette: 15.11.0

**Checks**
Site sécurisé: Non
URL de base côté serveur: Oui
Prêt pour LDAP: Non
Prêt pour OIDC: Non
Prêt pour OpenAI: Oui
E-mail configuré: Non


### Deployment

Docker (Synology)

### Additional Deployment Details

_No response_

--- a/mealie/scripts/reprocess_images.py
+++ b/mealie/scripts/reprocess_images.py
@@ -4,7 +4,7 @@ from concurrent.futures import ThreadPoolExecutor, as_completed
 from pathlib import Path
 
 import sqlalchemy as sa
-from PIL import Image
+from PIL import Image, UnidentifiedImageError
 from pydantic import UUID4
 
 from mealie.core import root_logger
@@ -84,7 +84,12 @@ def reprocess_recipe_images(recipe_id: UUID4, force_all: bool = False) -> None:
         image_file = service.dir_image / image_filename
         image_file.unlink(missing_ok=True)
 
-    service.minifier.minify(original_image, force=True)
+    try:
+        service.minifier.minify(original_image, force=True)
+    except UnidentifiedImageError:
+        pass  # source image is corrupted or invalid; skip
+    except Exception:
+        logger.exception(f"Failed to reprocess images for recipe {recipe_id}")
 
     # Reprocess timeline event images
     timeline_dir = service.dir_image_timeline

